src/main/java/com/fabriccommunity/spookytime/client/render/blockentity/CarvablePumpkinBER.java

```diff
@@ -0,0 +1,120 @@
+package com.fabriccommunity.spookytime.client.render.blockentity;
+
+import com.fabriccommunity.spookytime.block.entity.CarvablePumpkinBlockEntity;
+import com.mojang.blaze3d.platform.GlStateManager;
+
+import net.minecraft.block.Blocks;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.render.BufferBuilder;
+import net.minecraft.client.render.Tessellator;
+import net.minecraft.client.render.VertexFormats;
+import net.minecraft.client.render.block.entity.BlockEntityRenderer;
+import net.minecraft.client.texture.Sprite;
+import net.minecraft.client.texture.SpriteAtlasTexture;
+import net.minecraft.util.math.Direction;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * @author Indigo Amann
+ * @author cplir-c
+ */
+public class CarvablePumpkinBER extends BlockEntityRenderer<CarvablePumpkinBlockEntity> {
```

Comment:
I know it's long, but it would still be preferable to call the class `CarvablePumpkinBlockEntityRenderer`, or just `CarvablePumpkinRenderer`.